### DISCUSSION TITLE
Its possible set as default

### DISCUSSION THREAD

**Discussion by TheMrBigWill** [TheHerobrine/DarkerNodes#88] [Ideas]
every project need set once

**Comment by hugoattal** [comment]
@TheMrBigWill So, if you want Darker Nodes to be enabled by default, you can go here:
`D:\Program Files\Epic Games\UE_4.25\Engine\Plugins\Marketplace\DarkerNodes` (or your equivalent)
and then open the `DarkerNodes.uplugin` file, and add this line:
![image](https://user-images.githubusercontent.com/4563971/104447159-b59a1d00-559b-11eb-9ce5-a8766453da5e.png)
`"EnabledByDefault": true,`

I could add this line myself though, so everyone would have it activated by default 🤔... Wdyt?

**Reply by TheMrBigWill** [reply]
Its work！nice
tks！
